config/dev/common-dev.yml

```diff
@@ -1,5 +1,7 @@
 # This file contains only parameters that need to be overridden form the docker configuration
 operatorfabric:
+  customscreen:
+    dataFields: [ "stateName","message" ]
   rabbitmq:
```

Comment:
As an example put one field  :"keywords"
Add the field the defaultProcess card examples

Reply:
Done.